aioamqp/__init__.py

```diff
@@ -14,7 +15,8 @@
 @asyncio.coroutine
 def connect(host='localhost', port=None, login='guest', password='guest',
             virtualhost='/', ssl=False, login_method='AMQPLAIN', insist=False,
-            protocol_factory=AmqpProtocol, *, verify_ssl=True, loop=None, **kwargs):
+            protocol_factory=AmqpProtocol, *, verify_ssl=True, loop=None,
+            ssl_options={}, **kwargs):
```

Comment:
Using a `dict` as a default parameter is generally unsafe as it is mutable. However you seem to do no change to the dictionary in the function so it is probably ok here.